### PR TITLE
Remove Twistlock

### PR DESCRIPTION
**Pull request by ldraney**
# Description

This PR is to remove Twistlock from the repo in response to activating AWS Inspectors enhanced image scanning (also being activated today)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

From [this action](https://github.com/department-of-veterans-affairs/notification-api/actions/runs/5717286291)
You can see the dev-deploy was run without calling twistlock.
image
image

## Checklist

- [ ] My changes generate no new warnings

### CODE REVIEW

File: .github/workflows/twistlock.yml [hunk 1]
```diff
@@ -1,48 +0,0 @@
-name: Twistlock
```
Comment (cris-oddball) [inline .github/workflows/twistlock.yml:1]: I would not delete this just yet, there is a new ticket to do that for when we are assured AWS Inspector works.

File: README.md [hunk 2]
```diff
@@ -562,22 +562,6 @@ When adding environment variables to any `<filename>-task-definition.json` file,
 
 ---
 
-**Problem**: Github Actions is failing when running the 'Perform twistlock scan' step of the 'build-and-push' job
```
Comment (cris-oddball) [inline README.md:565]: Same, let's not remove this until we are 100% certain we don't need twistlock.

**Review by cris-oddball — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -34,7 +34,7 @@ install-safety:
 
 check-dependencies: install-safety ## Scan dependencies for security vulnerabilities
 	# The ignored issues are documented in requirements-app.txt.
-	safety check -r requirements.txt --full-report -i 51668
+	safety check -r requirements.txt --full-report -i 59234
```
Comment (cris-oddball) [inline Makefile:37]: Lucas, you need to leave 51668 in here because that is SQLAlchemy.

**Review by cris-oddball — CHANGES_REQUESTED** [review]

File: Makefile [hunk 4]
```diff
@@ -34,7 +34,7 @@ install-safety:
 
 check-dependencies: install-safety ## Scan dependencies for security vulnerabilities
 	# The ignored issues are documented in requirements-app.txt.
-	safety check -r requirements.txt --full-report -i 51668
+	safety check -r requirements.txt --full-report -i 51668 59234
```
Comment (cris-oddball) [inline Makefile:37]: the syntax is an -i in front of each vulnerability.

**Review by cris-oddball — CHANGES_REQUESTED** [review]

File: Makefile (hunk 4 above)
Comment (ldraney) [reply Makefile]: TY

**Review by cris-oddball — APPROVED** [review]

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -41,8 +41,8 @@ jobs:
           push: true
           tags: ${{ steps.login-ecr-vaec.outputs.registry }}/notification_api:${{ inputs.ref }}_${{ inputs.environment }}
 
-      - name: Dispatch Twistlock Workflow
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          gh workflow run -r master -F image-tag=${{ inputs.ref }}_${{ inputs.environment }} twistlock.yml
+      #- name: Dispatch Twistlock Workflow
```
Comment (k-macmillan) [inline .github/workflows/build.yml:44]: Please add a comment as to why this is disabled.

**Review by k-macmillan — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 5 above)
Comment (cris-oddball) [reply .github/workflows/build.yml:44]: Add the ticket number 1398 to that comment, please

**Review by cris-oddball — CHANGES_REQUESTED** [review]

**Review by cris-oddball — APPROVED** [review]

**Review by k-macmillan — APPROVED** [review]

**Pull request merged**